lib/userFactory.js

```diff
@@ -65,6 +65,38 @@ class UserFactory extends BaseFactory {
     }
 
     /**
+     * Get a user model by name, id, or token
+     * @method get
+     * @param   {Mixed}     config
+     * @param   {String}    [config.token]      Token of the user to look up
```

Comment:
Can we make it clear that the `token` here refers to `personal access token`. There is also `token` in the same file refering to the `github token` . Also the README please